docs/src/test-components-js.md

```diff
@@ -515,13 +569,33 @@ test('should work', async ({ mount }) => {
 
 ```js
 import { test, expect } from '@playwright/experimental-ct-solid';
-import HelloWorld from './HelloWorld';
+import { Greetings } from './Welcome';
```

Comment:
`HelloWorldComponent` looks ugly so decided to change `HelloWorld` to `Greetings`